PowershellTools.Common/Debugging/DebugEngineConstants.cs

```diff
@@ -46,7 +46,25 @@ public static class DebugEngineConstants
             @"\s+" + // spaces between function name and variable list
             @"\{([,\s]?[^\s,]+=\S+)*\}" + // list of parameters which are comma seperated, (note the equals sign)
             @"\s+([^\s:]+):" + // any number of spaces after the parameter list, and then the script name which ends right before a colon
-            @"\s+\w+ (\d)"; // any number of spaces and then some word such as "line" (depending on localization) and then a line number
+            @"\s+\w+ (\d)"; // any number of spaces (after the colon) and then the word "line" and then a number which denotes the line number
+
+        /// <summary>
+        /// Enumerates through all processes on a machine and looks for one running the powershell.exe module. Adds the process id and process name
+        /// of any valid processes to an array which is then returned.
+        /// </summary>
+        public const string EnumerateRemoteProcessesScript = @"

```

Comment:
Can we clean this up, or return it as something named, instead of this magic tuple?